sdk/communication/communication-messages-rest/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 2.1.0-beta.2 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
+## 3.0.0-beta.1 (2025-04-09)
```

Comment:
Why was it a major version bump if there's no breaking changes?

Reply:
Based on the guidance `YOU SHOULD increment the major version when making large feature changes` we are introducing a pretty large feature for Unified Messaging

Reply:
We were discussing this internally, I think we can relax that guideline a bit and continue to use semver concepts. In this case, if there is a non-breaking change that includes new features I would suggest 2.1.0. We will update the guidelines.

With that said, are there any breaking changes here? Or is it all additive?

Reply:
Thank you!

> @maorleger Its all additive, does this comment apply to all languages?

I think that each language architect has the authoritative say so you might check with them - with that said, I believe all languages that follow semver will provide the same guidance. My personal interpretation is that "large feature changes" in this case means "breaking feature (i.e. not bugfix) changes" and that the word "large" may be confusing. Thanks for bringing this up, we'll take a look at the guidelines 👍